mage_integrations/mage_integrations/tests/sources/api/test_api.py

```diff
@@ -216,6 +216,8 @@ def json_catalog_example():
 
 
 class ApiTest(unittest.TestCase):
+    maxDiff = None
```

Comment:
what's this for?

Reply:
It was for some unrelated test failures. The test failure was not giving complete diff. Removing this.